nixos/tests/packagekit.nix

```diff
@@ -0,0 +1,24 @@
+import ./make-test.nix ({ pkgs, ... }: {
```

Comment:
You need to add this to `all-tests.nix` to use it.

Reply:
Thanks